cypress/integration/ProcessorTest.ts

```diff
@@ -0,0 +1,342 @@
+
+      cy.ouiaId("item-0").within(() => {
+        cy.ouiaId("filter-key", "PF4/TextInput").type(filters[0][0]);
+        cy.ouiaId("filter-type", "PF4/FormSelect").select("String equals");
+      });
+      cy.ouiaId("item-0") //The filter-value was detached from DOM and we need to find the context again.
+        .ouiaId("filter-value", "PF4/TextInput")
+        .type(filters[0][2]);
+      cy.ouiaId("item-1").within(() => {
+        cy.ouiaId("filter-key", "PF4/TextInput").type(filters[1][0]);
+        cy.ouiaId("filter-type", "PF4/FormSelect").select("String equals");
+      });
+      cy.ouiaId("item-1") //The filter-value was detached from DOM and we need to find the context again.
+        .ouiaId("filter-value", "PF4/TextInput")
+        .type(filters[1][2]);
+
+      //Transformation
+      cy.ouiaId("transformation-description", "PF4/Text")
+        .parent()
+        .parent()
+        .find("textarea")
+        .should("be.visible")
+        .should("be.empty")
+        .type("I want to write here something.");
+
+      //Actions
+
+      cy.ouiaId("missing-actions", "PF4/TextInput")
+        .should("be.visible")
+        .should("be.disabled");
+      cy.ouiaId("action-type", "PF4/FormSelect")
+        .should("be.visible")
+        .select(action[0]);
+      cy.ouiaId("missing-actions", "PF4/TextInput").should("not.exist");
+      cy.ouiaId("channel", "PF4/TextInput")
+        .should("be.visible")
+        .type(action[1]);
+      cy.ouiaId("webhookUrl", "PF4/TextInput")
+        .should("be.visible")
+        .type(action[2]);
+
+      cy.ouiaId("submit").click();
+
+      cy.ouiaId("loading-table", "PF4/Card", { timeout: 30000 }).should(
+        "not.exist"
+      );
+
+      cy.ouiaId("Processors list table", "PF4/Table")
+        .ouiaId(processorName, "PF4/TableRow")
+        .should("be.visible")
+        .within(() => {
+          cy.get("td")
+            .eq(3)
+            .then(($state) => {
+              cy.wrap($state).should("have.text", "accepted");
+              cy.wrap($state, { timeout: 15000 }).should(
+                "have.text",
+                "provisioning"
+              );
+              cy.wrap($state, { timeout: 30000 }).should("have.text", "ready");
+            });
+          cy.get("td").eq(0).should("have.text", processorName).click();
+        });
+
+      cy.ouiaId("processor-name", "PF4/Text").should(
+        "have.text",
+        processorName
+      );
+
+      cy.ouiaId("filters", "PF4/Table").within(() => {
+        filters.forEach((item, rowIndex) => {
+          cy.ouiaId(item[0], "PF4/TableRow")
+            .find("td")
+            .each((cell, cellIndex) => {
+              cy.wrap(cell).should("have.text", filters[rowIndex][cellIndex]);
+            });
+        });
+      });
+
+      cy.get("code").should("have.text", "I want to write here something.");
+
+      cy.get("[class='pf-c-description-list__description']")
```

Comment:
`pf-c-description-list__description` elements contain `parameter values`. Would you mind if we check also `pf-c-description-list__term` elements, which should contain `parameter keys`?